types/office-js-preview/index.d.ts

```diff
@@ -18433,8 +18358,8 @@ declare namespace Office {
          *
          * @returns
          * An object that contains arrays of strings that match the regular expressions defined in the manifest XML file.
-         * The name of each array is equal to the corresponding value of the RegExName attribute of the matching `ItemHasRegularExpressionMatch` rule
-         * or the `FilterName` attribute of the matching `ItemHasKnownEntity` rule. For an `ItemHasRegularExpressionMatch` rule, a matching string has to occur in the property
+         * The name of each array is equal to the corresponding value of the RegExName attribute of the matching `ItemHasRegularExpressionMatch` rule.
+         * For an `ItemHasRegularExpressionMatch` rule, a matching string has to occur in the property
          * of the item that is specified by that rule. The `PropertyName` simple type defines the supported properties.
```

Comment:
here and other instances

```suggestion
         * of the item that's specified by that rule. The `PropertyName` simple type defines the supported properties.
```